assets/css/calypsoify.css

```diff
@@ -912,3 +912,31 @@ TODO: This can be removed after https://github.com/Automattic/jetpack/pull/10497
 #wpadminbar ul li#wp-admin-bar-notes.active > .ab-item {
 	background: #004967 !important;
 }
+
+/* Site title fix for long titles.
+TODO: This can be removed after https://github.com/Automattic/jetpack/pull/10507 is released */
+#calypso-sidebar-header ul li#calypso-sitename {
+	overflow: hidden;
+	white-space: nowrap;
+	width: 225px;
```

Comment:
I think we need to add a media query for this for <783px to `width: 150px`:

<img width="270" alt="screen shot 2018-11-01 at 4 07 07 pm" src="https://user-images.githubusercontent.com/10561050/47880158-7cc56180-ddf0-11e8-9283-1b89600e9478.png">